devops/girder/plugins/AnnotationPlugin/upenncontrast_annotation/server/models/datasetView.py

```diff
@@ -57,7 +57,7 @@ class DatasetViewSchema:
     }
 
 
-class DatasetView(ProxiedModel, AccessControlledModel):
+class DatasetView(AccessControlledModel, ProxiedModel):
```

Comment:
So here you are transforming the DatasetView, AnnotationProperty, ... models in AccessControlledModel instead of ProxiedModel.  
The difference is that now each document will handle its access control individually instead of relying on the parent document.  
I don't really see why you want to do that ?  
Instead, if you set the dataset as public, the underlying documents should become public as well